tests/test_imageset.py

```diff
@@ -5,18 +5,42 @@
 
 import pytest
 import six.moves.cPickle as pickle
+from mock import patch
```

Comment:
Given that you are only using one item from mock, I would probably prefer to just have:
```suggestion
import mock
```
and then use `mock.patch.object` below, which makes it clearer to anyone reading the code where `patch.object` comes from.

Reply:
Done